content/posts/September-2020.md

```diff
@@ -46,3 +46,15 @@ draft: false
   You can now use `test` job also on Pagure and GitLab thanks to that.
 
 [laura]: https://github.com/lbarcziova
+
+## Week 39 (September 21st - September 25th)
+
+- Service supports private namespaces now. They need to be configured in the service config. This is required in CentOS stream and not avaialble in GitHub. [packit-service#831](https://github.com/packit/packit-service/pull/831)
```

Comment:
That's not 100% correct. It's possible on GitLab and GitHub as well. It just allows selecting namespace where we work with private repositories. (The centos-stream namespace is here only a coincidence or our configuration.)

Reply:
my point here was that we are not enabling this in production for github so if anyone in the public reads this, they should know this is our internal thing and we still don't support private repos on github

Reply:
I'll open a PR.